#170283041 - Fixing Displaying a Single Request and the Menu Bar

nignanthomas opened the pull request:
#### What does this PR do?
Fixing Displaying a Single Request and the Menu Bar.

#### Description of Task to be completed?
- Pass the requestId as a props to the rendered component <SingleRequest /> in App.js
- Check for the routes to ignore the Menu Bar from <MenuComponent />

#### How should this be manually tested?
- Clone bg-fix-single-request-170283041 branch
- run `npm i`
- run `npm run dev`
- Click on `Requests` in the MenuBar
- Click on `View Request` to view a single request

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#170283041](https://www.pivotaltracker.com/story/show/170283041)

#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2019-12-16 at 13 05 33" src="https://user-images.githubusercontent.com/42959540/70902240-0344fe80-2005-11ea-9f7f-83c55e9f7c37.png">
<img width="349" alt="Screen Shot 2019-12-16 at 12 44 09" src="https://user-images.githubusercontent.com/42959540/70900744-17d3c780-2002-11ea-808f-12c3dea88691.png">
<img width="347" alt="Screen Shot 2019-12-16 at 13 06 31" src="https://user-images.githubusercontent.com/42959540/70902255-0a6c0c80-2005-11ea-89b9-f9c681a7cec3.png">

#### Questions:
N/A

nignanthomas committed:
bug-fix(single-request&menu): Fixing Displaying a Single Request and the Menu Bar

- pass the requestId as a props to the rendered component <SingleRequest /> in App.js

- check for the routes to ignore the Menu Bar from <MenuComponent />

[Finishes #170283041]

alainmateso commented:
Impressive work @nignanthomas , Kindly work on the following feedback:
- It would be better to remove the logout button from the menubar since it does not remove the token from the storage at least and if it is to be there, it is supposed to be under the account icon.
- I believe it would be better as well if the fields are not editable when viewing a single request. I don't see the need of letting the user manipulate the data when they are not going to edit and send it.
- Last but not least, I've just realized that the booking box, doesn't retrieve anything, kindly remove it

Other than that, this is awesome.

alainmateso reviewed (changes requested):
Impressive work @nignanthomas , Kindly work on the following feedback:
- It would be better to remove the logout button from the menubar since it does not remove the token from the storage at least and if it is to be there, it is supposed to be under the account icon.
- I believe it would be better as well if the fields are not editable when viewing a single request. I don't see the need of letting the user manipulate the data when they are not going to edit and send it.
- Last but not least, I've just realized that the booking box, doesn't retrieve anything, kindly remove it

Other than that, this is awesome.

nignanthomas committed:
bug-fix(single-requet&menu): Fixing Displaying a Single Request and the Menu Bar

- pass the requestId as a props to the rendered component <SingleRequest /> in App.js

- check for the routes to ignore the Menu Bar from <MenuComponent />

[Finishes #170283041]

nignanthomas committed:
bug-fix(single-request&menu): Fixing Displaying a Single Request and the Menu Bar

- pass the requestId as a props to the rendered component <SingleRequest /> in App.js

- check for the routes to ignore the Menu Bar from <MenuComponent />

[Finishes #170283041]
